0.pre-processing/README.md

```diff
@@ -0,0 +1,7 @@
+# pre-processing
+
+This folder contains script to reproduce steps needed to initialise and prepare the cytodata 2019 challenge.
+
+The pseudonymisation is not reproducible as the codes are randomly generated each time.
```

Comment:
This is an important disclaimer. It's definitely ok for our purposes here (especially since the data are archived), but for other applications, [setting a seed](https://stackoverflow.com/questions/13605271/reasons-for-using-the-set-seed-function) is very important.

Reply:
also this time Ive set a seed just for linting the code